### DISCUSSION TITLE
Student Pack Copilot

### DISCUSSION THREAD

**Discussion by OmarOcasio20366** [community/community#31965] [GitHub Education]
Hello, I currently have the student pack and everytime I start to write code the copilot extension deactivates. Im using VS Code, I've re-installed the extension and synced my account and this is the error message that outputs: 

[INFO] [default] [2022-09-04T21:46:41.462Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-09-04T21:46:41.880Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 413 ms
[INFO] [fetch] [2022-09-04T21:46:41.881Z] Please upgrade your Copilot extension to continue using this service.

[INFO] [default] [2022-09-04T21:46:43.053Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-09-04T21:46:43.148Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 95 ms
[INFO] [fetch] [2022-09-04T21:46:43.149Z] Please upgrade your Copilot extension to continue using this service.

**Comment by rfals** [comment]
Same here, did u manage to fix it?
